sdcm/nemesis.py

```diff
@@ -1775,29 +1775,31 @@ def disrupt_mgmt_backup(self):
         self._mgmt_backup(backup_specific_tables=False)
 
     def _mgmt_backup(self, backup_specific_tables):
```

Comment:
you can override the method via the Sirenada repo
Go to the `sct_plugin/nemesis_cloud.py` file and create this function inside the file
```
    def _mgmt_backup(self, backup_specific_tables):
       the nee logic
sdcm.nemesis._mgmt_backup = _mgmt_backup
```

Reply:
right, i just didn't want to duplicate all logic and code from SCT. better it will all be found at one place as added for k8s.